Documentation/tls/tls-certificates.md

```diff
@@ -0,0 +1,49 @@
+# Transport Layer Security (TLS) Certificates
+
+Tectonic secures all communications between cluster components. At install time, certificates can be provided in the following ways, from most automated to most customized:
+
+1. Installer generates a new CA, intermediate CAs and all certificates (default)
+2. Installer uses your provided CA (or intermediate) to generate all certificates
+3. Certificates are generated out-of-band (guides below) and provided to the Installer
+
+Read about the [TLS topology][tls-topology] for more technical information about the intermediate CAs used between components.
+
+## Components and Certificate Sources
+
+| Type of Traffic | Source of Certificate |
+|:----------|:----------------------|
+| API communication (external and internal) | Generated (default) </br> [User-provided][tls-identity] |
```

Comment:
This is tls between the console the apiserver and the identity pods. This is pretty specific to the way users authenticate with the cluster. This isn't related to how the apiserver communicates with anything. Just how the users authenticate to the apiserver itself.

Reply:
then it's redundant to 'Using the Tectonic Console and Ingress' below, and should be removed? or rewritten to 'User authentication with the cluster (Console, API Server, and Identity Pods)'

Reply:
User Authentication makes more sense to me